GDevelop.js/types/gdbehaviormetadata.js

```diff
@@ -25,6 +25,7 @@ declare class gdBehaviorMetadata {
   getAllStrExpressions(): gdMapStringExpressionMetadata;
   setIncludeFile(includeFile: string): gdBehaviorMetadata;
   addIncludeFile(includeFile: string): gdBehaviorMetadata;
+  addRequiredFile(resourceFile: string): gdBehaviorMetadata;
```

Comment:
I only added a "add" method because I think that there is no need to rewrite the list.
Why does the "set" exist?

Reply:
I think we should just have been using addIncludeFile, wrong past decision.